src/App.jsx

```diff
@@ -17,22 +17,91 @@ const App = () => {
   const [todoList, setTodoList] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
 
+  /* Async function */
+  const fetchData = async () => {
+    const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}?sort%5B0%5D%5Bfield%5D=createdAt&sort%5B0%5D%5Bdirection%5D=asc`;
+    const options = {
+      method:"GET",
+      headers: {
+        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`
+      }
+    };
+    try{
+      const response = await fetch(url, options);
+      if (!response.ok){
+        throw new Error(`${response.status}`);
+      }
+      const data = await response.json();
+      const todos = data.records.map((todo) => {
+        return {
+          title: todo.fields.title,
+          id:todo.id
+        };
+      });
+      console.log(todos);
+      setTodoList(todos);
+      setIsLoading(false);
+    }catch(error){
+      console.log(`Error fetching data: ${error}`)
+    }
+  };
+
+  const postData = async({title, createdAt}) => {
+    const airtableData = {
+      fields: {
+        title: title,
+        createdAt: createdAt
+      }
+    };
+    const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}\\`;
+    const options = {
+      method: "POST",
+      headers: {
+        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(airtableData)
+    };
+    
+    try{
+      const response = await fetch(url, options);
+      if(!response.ok){
+        throw new Error(`${response.status}`);
+      }
+      const data = await response.json();
+      const newTodo = {
+        title: data.fields.title,
+        id: data.id
+      }
+      return newTodo;
+    } catch(error){
+      console.log(`Error: ${error}`);
+    }
+  };
+
+  const deleteData = async(id) => {
+    const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}/${id}\\`;
+    const options = {
+      method: "DELETE",
+      headers: {
+        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`
+      }
+    };
+    try{
+      const response = await fetch(url, options);
+      if(!response.ok){
+        throw new Error(`${response.status}`);
+      }
+      const data = await response.json();
+      return data;
+    } catch(error){
+      console.log(`Error: ${error}`);
+    }
+  };
```

Comment:
Awesome work! 💯 🥇 
I could not figure out why the item reappears for a moment after is deleted. Let me know if you figure it out! 😃